### DISCUSSION TITLE
How to just run regtest and output data after just 1 (overall) timestep?

### DISCUSSION THREAD

**Discussion by japham0** [NOAA-EMC/WW3#1245] [Q&A]
I am a bit confused about the time stepping-  

I have been playing with the regtest ww3_tp2.2, where all 4 of the 4 input timesteps are 2200. I am having trouble figuring out how this relates to the total number of timesteps the model solves for, as well as what this means in relation to the strides.

1. Since 2200 seconds is ~36.67 minutes, does this mean that if I change in the   ww3_shel.nml file:

  DOMAIN%STOP    = '20220606 003700'

Then the output data (say HS) will be after the solver or numerical scheme iterates over just 1 timestep? 

2. What does the stride mean in relation to the timestep? 
This is from the ww3_ounf.nml : 
! -------------------------------------------------------------------- !
! Define the output fields to postprocess via FIELD_NML namelist
! -------------------------------------------------------------------- !
&FIELD_NML
  FIELD%TIMESTART        =  '20220606 000000'
  FIELD%TIMESTRIDE       =  '10800.'
  FIELD%TIMECOUNT        =  '100'
 
what does 10,800 mean in relation to the timestep, since 10,800 is not divisible by 2200?

**Comment by JessicaMeixner-NOAA** [comment]
Some helpful information on time steps can be found here: https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#cfl-condition

**Reply by japham0** [reply]
Hello, thank you for the link, however I understand the CFL condition and am not running into an error regarding that, since I did not change the timestep values. 

By default in the regtest, the timesteps were provided to all be 2200. I just wanted to ask regarding the output data, how would I just output data after iterating over just 1 timestep? And as I asked above, how does 10,800 for stride relate to the timestep? I am not running into any errors running it, I just want to know what stride means in relation to the timestep.
